xios_examples/shared_testing.py

```diff
@@ -21,6 +21,19 @@ class _TestCase(unittest.TestCase):
     transient_outputs = []
     rtol = 5e-03
     executable = './resample.exe'
+    mesh_file = None
+
+    @classmethod
+    def make_netcdf(cls, inf, inputfile, nc_method='cdl_files'):
+        if nc_method == 'cdl_files':
+            # create a netCDF file from the `.cdl` input
+            subprocess.run(['ncgen', '-k', 'nc4', '-o', inputfile,
+                            inf], cwd=cls.test_dir, check=True)
+        elif nc_method == 'data_func':
+            # create a  netCDF file from an analytic function
+            os.chdir(cls.test_dir)
+            import xios_examples.gen_netcdf as gn
```

Comment:
please may we keep `import`s at the top of the module?

Reply:
moved as requested